src/content/1.7/development/architecture/cqrs.md

```diff
@@ -0,0 +1,69 @@
+---
+title: CQRS
+weight: 10
+---
+
+# CQRS
+
+## What is CQRS?
+
+CQRS stands for _Command Query Responsibility Segregation_. In brief, the CQRS pattern suggests to separate "write" and "read" models, which it refers to as Commands and Queries.
+
+## Why using CQRS in PrestaShop?
+
```

Comment:
Can you explain why we need to keep core and why we need to isolate it ?
Something like

"We are currently migrating PrestaShop Back Office to Symfony but the new Controllers, Templates and classes dedicated to the BO are still using the Core classes of PrestaShop used by the previous BO. One day, we will remove this Core too. Until that day, we need to allo the new BO to access these classes and data while keeping a clear border between these 2 areas of code as one is expected to remain and the other one to disappear."

Reply:
Do you think this commit https://github.com/PrestaShop/docs/pull/232/commits/b908ef4eff4effd4a9170531bd0ebe171f3f5c5c explains it?